visualization/app/codeCharta/ui/codeMap/codeMap.component.spec.ts

```diff
@@ -31,18 +31,15 @@ describe("ColorSettingsPanelController", () => {
 	}
 
 	function mockElement() {
-		$element = jest.fn<Element>(() => {
-			return [
+		$element = {} as any
+		$element = Object.assign($element, {
+			children: [
 				{
-					children: [
-						{
-							clientWidth: 50,
-							clientHeight: 100
-						}
-					]
+					clientWidth: 50,
+					clientHeight: 100
 				}
 			]
-		})()
+		})
```

Comment:
```suggestion
		} as Record<string, unknown>
```

Reply:
I don't really understand that suggestions. Please take another look at the line, since I changed it with another previous suggestion.

Reply:
It belonged to another suggestion. The suggestion range would just not allow to do it in one comment :-)